app/experimenter/experiments/migrations/0047_auto_20190422_1819.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.1.7 on 2019-04-22 18:19
```

Comment:
Yep exactly, but you don't have to do it in a separate migration file, just move all this to 46 and then in the operations at the bottom you put them in order

    operations = [
        migrations.AddField(....
        migrations.RunPython(....
    ]